pkg/mapper/iac-providers/cft/config/lambda-function.go

```diff
@@ -55,49 +54,102 @@ type LambdaFunctionConfig struct {
 }
 
 // GetLambdaFunctionConfig returns config for LambdaFunction
-func GetLambdaFunctionConfig(f *lambda.Function) []AWSResourceConfig {
+func GetLambdaFunctionConfig(sf interface{}) []AWSResourceConfig {
+	if l, ok := sf.(*lambda.Function); ok {
+		return getLambdaConfig(l)
+	}
+
+	if s, ok := sf.(*serverless.Function); ok {
+		return getServerlessConfig(s)
+	}
```

Comment:
if there are only two types that are going to call this function, we can re-write the function body as below:

```
if l, ok := sf.(*lambda.Function); ok {
  return getLambdaConfig(l)
}
return getServerlessConfig(s)
```